hazelcast/src/test/java/com/hazelcast/map/impl/eviction/ExpirationManagerTimeoutTest.java

```diff
@@ -42,22 +39,18 @@
     public void afterShortExpirationEntryShouldBeAway() throws InterruptedException {
         final String KEY = "key";
 
-        Config hConfig = new Config()
-                .addMapConfig(new MapConfig().setName("test")
-                        .setMaxSizeConfig(new MaxSizeConfig(200, MaxSizeConfig.MaxSizePolicy.FREE_HEAP_SIZE))
-                        .setTimeToLiveSeconds(20));
-        final HazelcastInstance node = createHazelcastInstance(hConfig);
+        final HazelcastInstance node = createHazelcastInstance();
         try {
             IMap<String, String> map = node.getMap("test");
             // after 1 second entry should be evicted
-            map.put(KEY, "value", 1, TimeUnit.SECONDS);
+            map.set(KEY, "value", 1500, TimeUnit.MILLISECONDS);
             // short time after adding it to the map, all ok
             map.lock(KEY);
             Object object = map.get(KEY);
             map.unlock(KEY);
             assertNotNull(object);
 
-            Thread.sleep(1200);
+            Thread.sleep(1700);
```

Comment:
maybe use `sleepAtLeastMillis` to ensure sleep time?

Reply:
done